src/app/conf/2025/register-today/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { clsx } from "clsx"
+import NextImage from "next-image-export-optimizer"
+
+import { Button } from "../../_design-system/button"
+
+import speakerImage from "./speaker.webp"
+
+export interface RegisterTodayProps {
+  className?: string
+}
+
+export function RegisterToday({ className }: RegisterTodayProps) {
+  return (
+    <section
+      className={clsx(
+        "flex gap-10 px-4 max-lg:flex-col-reverse lg:px-12 xl:gap-x-24 xl:px-24",
+        className,
+      )}
+    >
+      {/* todo: test if the placeholder works in deploy preview */}
```

Comment:
This is something we'll need to test because blur placeholders just don't work for me locally, and yet they seem to work in prod.